README.md

```diff
@@ -27,6 +27,8 @@ spring.datasource.driver-class-name = org.mariadb.jdbc.Driver
 spring.jpa.hibernate.ddl-auto = create-drop
 spring.security.user.name = # username
 spring.security.user.password = # user password
+jwt.ExpirationMs = # login token expiration time in miliseconds
```

Comment:
Previously I pointed you that we shouldn't change arbitary name of properties. You didn't fix it. I wrote that "
Why did you change that? Leave it as it is. It is dsrm specific so i believe the prefix dsrm was good idea.
But if you want you can change this to dsrm.auth.jwt.expirationMs and dsrm.auth.jwt.refreshExpirationMs. It would be slightly better" Please fix it.